src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
+const MAX_HEALTHS = 100;
+const BITE_DAMAGE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name) {
+    this.health = MAX_HEALTHS;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  static removeDead() {
+    Animal.alive = Animal.alive.filter((animal) => animal.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name) {
```

Comment:
The `Herbivore` constructor takes a `health` parameter, but it is not used. Consider removing it or using it to initialize the `health` property.

Reply:
> The `Herbivore` constructor takes a `health` parameter, but it is not used. Consider removing it or using it to initialize the `health` property.

Good point, and it's invalid, it should be the same order as in the parent class.